util/perl/OpenSSL/paramnames.pm

```diff
@@ -420,6 +419,7 @@ my %params = (
 
 # ML-KEM parameters
     'PKEY_PARAM_ML_KEM_SEED' => "seed",
+    'PKEY_PARAM_ML_KEM_RETAIN_SEED' => "ml-kem/retain_seed",
```

Comment:
Use of `/` is awkward here. I would use just `ml-kem-retain_seed` if this is really supposed to be ml-kem specific. We have various `rsa-something` params for example.

Reply:
Now using `.` which matches placement of the parameter in an `ml-kem` subsection of the provider config by (shareable across providers) reference.